### PR TITLE
Adopt (stop tracking) certs from tripleo

### PR DESCRIPTION
**Pull request by xek**
Adopt (stop tracking) certs from tripleo

https://issues.redhat.com/browse/OSPRH-6619

### CODE REVIEW

File: roles/edpm_install_certs/tasks/main.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Adopt (stop tracking) certs from tripleo
+  ansible.builtin.include_tasks: adoption.yml
```
Comment (slagle) [inline roles/edpm_install_certs/tasks/main.yml:18]: i don't think this is something we want included every time, even for greenfield deployments. Could we include it from the `edpm_tripleo_cleanup` role instead?

Comment (xek) [reply roles/edpm_install_certs/tasks/main.yml]: Yeah, makes sense, I moved it there.

**Review by slagle — COMMENTED** [review]
+2

**Comment by xek** [comment]
recheck error: timed out waiting for the condition on openstackdataplanenodesets/openstack-edpm-ipam

**Comment by xek** [comment]
recheck https://github.com/openstack-k8s-operators/ci-framework/pull/1815 merged

**Comment by xek** [comment]
recheck

**Comment by xek** [comment]
recheck

**Review by bogdando — COMMENTED** [review]
lgtm

**Comment by xek** [comment]
I pushed ansible lint fixes

**Comment by xek** [comment]
recheck

**Review by fao89 — APPROVED** [review]

**Pull request merged**